package.json

```diff
@@ -66,7 +66,7 @@
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
-    "node-sass": "^7.0.3",
+    "node-sass": "7.x 8.x",
```

Comment:
is this a valid format? it can be tested here: https://semver.npmjs.com

I think something like the following should work: 

```suggestion
    "node-sass": "7.0.3 - 8",
```

Reply:
Actually we can use the ```^7.0.0 || ^8.0.0``` format that works well, good for you ?

Reply:
As in ```^7.0.3 || ^8.0.0```